File: packages/server/ExchangeGame/GameObjects/Attendee.cs
using Microsoft.Extensions.Logging;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExchangeGame
{
    public class Attendee: GameObject
    {
        public bool InCall { get; set; }

        public Player Player { get; private set; }


        public Attendee(string displayName, Player player) : base(displayName)
        {
            Player = player;
            _logger.LogInformation($"Person {DisplayName} is in the game");
        }
    }
}

File: packages/server-v2/src/GameObjects/Attendee.ts
import { generateId } from "../utils";

export default class Attendee {
    public inCall: boolean = false;
    public playerId: number;
    public id: number;
    public displayName: string;

    constructor(displayName: string, playerId: number) {
        this.displayName = displayName;
        this.id = generateId();
        this.playerId = playerId;
        console.log(`Person ${displayName} is in the game`);
    }
}